homeassistant/components/light/tradfri.py

```diff
@@ -107,10 +107,18 @@ def __init__(self, light, api):
 
         if self._light_data.hex_color is not None:
             if self._light.device_info.manufacturer == IKEA:
-                self._features |= SUPPORT_COLOR_TEMP
+                if "CWS" in self._light.device_info.model_number:
+                    self._features |= SUPPORT_RGB_COLOR
+                else:
+                    self._features |= SUPPORT_COLOR_TEMP
             else:
                 self._features |= SUPPORT_RGB_COLOR
 
+        _LOGGER.debug(
+            "features detected for %s %s: %d",
+            self._name, self._light.device_info.model_number, self._features
+        )
```

Comment:
You see the feature on UI, there need no debug log...

Reply:
removed logging of detected features